link/kprobe.go

```diff
@@ -161,7 +161,7 @@ func kprobe(symbol string, prog *ebpf.Program, opts *KprobeOptions, ret bool) (*
 
 	args := probeArgs{
 		pid:    perfAllThreads,
-		symbol: platformPrefix(symbol),
+		symbol: symbol,
```

Comment:
Nit: please add the justification you gave in the PR description to the commit message.